posthog/models/organization.py

```diff
@@ -372,3 +373,13 @@ def ensure_organization_membership_consistency(sender, instance: OrganizationMem
         save_user = True
     if save_user:
         instance.user.save()
+
+
+def is_feature_available(organization_id: Optional[UUID], feature: AvailableFeature):
+    if organization_id is not None:
```

Comment:
should be if it _is_ None ?
```suggestion
    if organization_id is None:
```

Reply:
hah maybe!